packages/react-lowcode/src/codegen/tests/api/api.test.ts

```diff
@@ -11,8 +10,15 @@ describe(".api tests", () => {
         const file = fs.readFileSync(path.resolve('src\\codegen\\tests\\list\\files\\is-datatable-test-file.txt'),'utf-8') 
   
         const source : SourceLineCol = {lineNumber: 12, columnNumber:17, fileName:'test'}
-        isDataTableWidget(file, source)
-      }); 
+        isSelectedDataTable(file, source)
+    }); 
+
+    test(".is form widget", () => {
+        const file = fs.readFileSync(path.resolve('src\\codegen\\tests\\detail\\detail-test-file.txt'),'utf-8') 
```

Comment:
Is it possible to use forward slashes here (even on windows)?

Reply:
Yes you are right..we will use forward slashes from now.